phac-nml/irida-next
# Prettier updates

## What does this PR do and why?
_Describe in detail what your merge request does and why._

1. Updated to the recent prettier and prettier-plugin-tailwind 
2. Remove very outdated babel configuration file

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Ensure recommended extensions are installed.  In this case, ensure "esbenp.prettier-vscode" extension is installed.
2. Reload the vs code window (`ctrl-p` search `reload window`)
3. Run `pnpm install`

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
